README.md

```diff
@@ -101,7 +101,12 @@ as a compilation tool on top of Go:
 
     $ go get -u github.com/mitchellh/gox
 
-Next, clone this repository into `$GOPATH/src/github.com/mitchellh/packer`.
+Next, clone this repository into `$GOPATH/src/github.com/mitchellh/packer`:
```

Comment:
We should change this to `go get github.com/mitchellh/packer` which will put it in the right place.